### ISSUE TITLE
Check official ECO Docker image

### ISSUE THREAD

**Issue by zokradonh** [zokradonh/ecogameserver#3]
Check if the official ECO Docker image works like this or better and deprecate this image if needed.

https://hub.docker.com/r/strangeloopgames/eco-game-server

**Comment by Skaronator** [comment]
Looks like they doesn't have a stable tag pinned at the major version. (Like "0.8" which always contain the latest fixes) And I couldn't see the latest tag pinned to the last stable release.

**Comment by zokradonh** [comment]
Yes, and I just checked the latest beta 9.0.0 tag. They use .NET Core instead of Mono.
If that is true, that the ECO Server is runnable with .NET Core, that would simplify and improve linux server.

This looks more like a helper image for the developers debugging a new multi-platform server. There are debug and some (minor) source code files included in the image.

**Comment by ymarillet** [comment]
any plans to make this repo 9.0 compatible ?

**Comment by zokradonh** [comment]
I think we can deprecate this repo right now.
`strangeloopgames/eco-game-server:0.9.0.1-beta` looks like a fully functional .NET Core server. Not tested yet though.

**Comment by zokradonh** [comment]
Just tested. 
The official docker image is perfectly fine, though tag `latest` maybe something else.

https://hub.docker.com/r/strangeloopgames/eco-game-server/tags

Some good to know directories:
/app/Configs
/app/Storage
/app/DefaultWorld
/app/Mods
/app/WebClient

Ports:
3000/udp
3001/tcp

All in all you can just replace the image name. Handling of their image is the same like my image.
For the test I just replaced
`image: zokradonh/ecogameserver`
with
`image: strangeloopgames/eco-game-server:0.9.0.1-beta`
and was able to start my old 8.3.3 world and old eco-config with this new image without any other change.
Well, in the start logs it said something about old config version replaced by new.

**Comment by jeliasson** [comment]
> 
> 
> Just tested.
> The official docker image is perfectly fine, though tag `latest` maybe something else.
> 

Yeah, tag `latest` is something else for some reason. Tag `default` seem to pick up the `0.9.0.1-beta` build.

Still used your image to build out the initial configs templates tho. 😅

**Comment by zokradonh** [comment]
`latest` is 0.9.0.0-beta 
`default` is 0.9.0.1-beta

Right now. But I think they will change that and make `latest` the way to go.

> Still used your image to build out the initial configs templates tho. 😅

Hehe, changing my image to 9.0 would require some changes since we can drop `mono` and take a simple `debian` base image.
Also, we need to download EcoLinuxServer.zip instead of EcoServer.zip

I will not change this, but contribution is welcome if someone wants to do this.

**Comment by ymarillet** [comment]
official image is working fine for me after struggling a bit with config files / world size / strange exceptions
